### PR TITLE
seo: Add title to all docs pages

### PR DESCRIPTION
**Pull request by ZJvandeWeg**

### CODE REVIEW

**Comment by joepavitt** [comment]
Similar to the equivalent Handbook PR, ideally this uses `navTitle` or `meta.title` instead as discussed

**Comment by ZJvandeWeg** [comment]
@joepavitt This should be ready for review and merging.

File: docs/contribute/workflows/README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+title: State Flows
```
Comment (joepavitt) [inline docs/contribute/workflows/README.md:2]:
```suggestion
navTitle: State Flows
```

File: docs/contribute/workflows/signup.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+---
+title: User Sign up Flow
```
Comment (joepavitt) [inline docs/contribute/workflows/signup.md:2]:
```suggestion
navTitle: User Sign up Flow
```

File: docs/contribute/workflows/team-create.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+---
+title: Team creation Flow
```
Comment (joepavitt) [inline docs/contribute/workflows/team-create.md:2]:
```suggestion
navTitle: Team creation Flow
```

**Review by joepavitt — CHANGES_REQUESTED** [review]

**Comment by joepavitt** [comment]
Given the release tomorrow, is there much merit in the backport label?

**Pull request merged**